lib/database/migrations/20240716134539-add-default-GAQ-detectors.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+    up: async (queryInterface) => queryInterface.sequelize.transaction(async (transaction) => {
+        await queryInterface.sequelize.query(`
+            INSERT INTO global_aggregated_quality_detectors(data_pass_id, run_number, dpl_detector_id)
+            SELECT dpr.data_pass_id, r.run_number, dpl.id
+                FROM data_passes_runs AS dpr
+                INNER JOIN runs AS r
+                    ON r.run_number = dpr.run_number
+                    AND r.pdp_beam_type = 'pp'
```

Comment:
Please extract the differences of this query in const variables.
Currently it is very difficult to read and quickly see the differences between the two such as beam type and detectors

Reply:
refactored

Reply:
Given that you pass as parameters, it would be nice to use the `replacements` field of sequelize when passing parameters. 
While in this scenario there is no user input, it would be good to have it as standard

Reply:
applied